### PR TITLE
[stable10] use samaccountname from session for wnd login (resend)

### PR DESCRIPTION
**Pull request by PVince81**
already approved, waiting for CI before merge

### CODE REVIEW

**Comment by PVince81** [comment]
how come php-cs-fixer failed here when it passed in the master PR ?? https://github.com/owncloud/core/pull/31951

File: tests/lib/User/SyncServiceTest.php [hunk 1]
```diff
@@ -300,5 +300,4 @@ public function testSyncUserName() {
 		$s = new SyncService($this->config, $this->logger, $this->mapper);
 		static::invokePrivate($s, 'syncUserName', [$a, $backend]);
 	}
-
```
Comment (phil-davis) [inline tests/lib/User/SyncServiceTest.php]: I remember that a newer version of php-cs-fixer started finding more blank lines.
Maybe the master PR is still rebased to an old point off master. Merging a PR like that might cause master to start failing.
@PVince81

Comment (PVince81) [reply tests/lib/User/SyncServiceTest.php]: @phil-davis I checked the master PR as is and ran the style checker. No issues.
Then rebased onto master, also no issues.

Maybe this happens from times to times when git automatically resolves conflicts.

**Comment by PVince81** [comment]
weird failures:
```
    /drone/src/tests/acceptance/features/apiTrashbin/trashbin-new-endpoint.feature:190
    /drone/src/tests/acceptance/features/apiTrashbin/trashbin-new-endpoint.feature:201
    /drone/src/tests/acceptance/features/apiTrashbin/trashbin-old-endpoint.feature:190
    /drone/src/tests/acceptance/features/apiTrashbin/trashbin-old-endpoint.feature:201
/drone/src/tests/acceptance/features/apiTrashbin/trashbin-old-endpoint.feature:219
```

restarted this job, just in case...

**Comment by PVince81** [comment]
now external storage:
```

--- Failed scenarios:

    /drone/src/tests/acceptance/features/apiMain/checksums.feature:187
    /drone/src/tests/acceptance/features/apiMain/checksums.feature:188
    /drone/src/tests/acceptance/features/apiMain/external-storage.feature:8
    /drone/src/tests/acceptance/features/apiMain/external-storage.feature:26
    /drone/src/tests/acceptance/features/apiMain/external-storage.feature:35
    /drone/src/tests/acceptance/features/apiMain/external-storage.feature:45
/drone/src/tests/acceptance/features/apiMain/external-storage.feature:55
```

need to be careful here then as the change in this PR actually affects external storage.

**Comment by PVince81** [comment]
I have these failures locally:
```
    /srv/www/htdocs/owncloud/tests/acceptance/features/apiMain/external-storage.feature:8
    /srv/www/htdocs/owncloud/tests/acceptance/features/apiMain/external-storage.feature:26
    /srv/www/htdocs/owncloud/tests/acceptance/features/apiMain/external-storage.feature:35
    /srv/www/htdocs/owncloud/tests/acceptance/features/apiMain/external-storage.feature:45
```

will compare with stable10 just in case

**Comment by PVince81** [comment]
oh noes, these also fail on stable10 for some reason :cry:

**Comment by PVince81** [comment]
engaging bisect engines

**Comment by PVince81** [comment]
v10.0.9 runs fine

**Pull request merged**